web/src/main/java/com/es/phoneshop/web/controller/pages/ProductDetailsPageController.java

```diff
@@ -24,6 +24,7 @@
     public String showProductDetails(@PathVariable long phoneId, Model model) {
         model.addAttribute("cartTotal", cartService.getCart().getTotal());
         model.addAttribute("phone", productService.getPhone(phoneId));
+        model.addAttribute("count", cartService.countProducts());
```

Comment:
use `ControllerAdvice` in order to avoid setting 'count' model attribute in multiple controllers